lib/product/counter_hello_button.dart

```diff
@@ -0,0 +1,32 @@
+// ignore_for_file: unused_import
+
+import 'package:flutter/material.dart';
+
+class CounterHelloButton extends StatefulWidget {
+  const CounterHelloButton({super.key});
+
+  @override
+  State<CounterHelloButton> createState() => _CounterHelloButtonState();
+}
+
+class _CounterHelloButtonState extends State<CounterHelloButton> {
+  int _counterCustom = 0;
+  final String _wellcomeTitle = "Merhaba";
+  void _updateCounter() {
```

Comment:
Add space between variables and methods.

```suggestion
  final String _wellcomeTitle = "Merhaba";
  
  void _updateCounter() {
```

Reply:
Done.